bin/install-qa-check.d/95xattr

```diff
@@ -0,0 +1,54 @@
+# Check for xattrs.
```

Comment:
Can you include a bit more information/background here? (Similar to floppym's [query](https://archives.gentoo.org/gentoo-portage-dev/message/67a537a1e97692db1f9903fb003b0bbd) on the ML).